SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp

```diff
@@ -36,22 +36,30 @@ void FSpatialGDKEditor::GenerateSchema(FSimpleDelegate SuccessCallback, FSimpleD
 
 	PreProcessSchemaMap();
 
+	// Compile all dirty blueprints
+	TArray<UBlueprint*> ErroredBlueprints;
+	bool PromptForCompilation = false;
+	UEditorEngine::ResolveDirtyBlueprints(PromptForCompilation, ErroredBlueprints);
+
+	if (ErroredBlueprints.Num() > 0)
+	{
+		UE_LOG(LogSpatialGDKEditor, Error, TEXT("There were errors when compiling blueprints. Schema has not been generated."));
+		FailureCallback.ExecuteIfBound();
+		GeneralProjectSettings->bSpatialNetworking = bCachedSpatialNetworking;
```

Comment:
Curious why this line is needed?

Reply:
Not 100% sure, but it would have always been called without my changes (see [here](https://github.com/spatialos/UnrealGDK/blob/14b3a98f5b13f0fbbc9c97a0f0047dcdc778bd25/SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp#L64)), so I didn't want to accidentally break anything. Let me know if you think it is safe to take it out.